search/src/Components/upload/upload_stepper.js

```diff
@@ -206,42 +27,80 @@ const useStyles = (theme) => ({
   },
 });
 
-function getSteps() {
-  return ['Upload Weedcoco', 'Add Agcontext', 'Upload Images'];
+function getSteps(upload_type) {
+  return upload_type === 'coco'?
+         ['Upload Coco', 'Add Agcontext', 'Upload Images']:
```

Comment:
We should work towards this capitalisation in the UI:
```suggestion
         ['Upload COCO', 'Add AgContext', 'Upload Images']:
```